### PR TITLE
Toctree & Formatting issues fixed

### PR DESCRIPTION
**Pull request by nidhibhammar**
## What Changed?

Toctree & Formatting issues fixed. Also added the missing table.

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**
- [X] This PR adds new content
- [X] This PR changes existing content
- [ ] This PR removes existing content

### CODE REVIEW

File: product_docs/docs/pem/8.0/pem_online_help/04_toc_pem_features/12_pem_manage_probes/03_pem_probe_config/index.mdx [hunk 1]
```diff
@@ -10,23 +10,19 @@ A [probe](01_pem_probes/#pem_probes) is a scheduled task that returns a set of p
 
 The `Manage Probes` tab provides a set of `Quick Links` that you can use to create and manage probes:
 
-> -   Click the `` `Manage Custom Probes `` &lt;pem_custom_probes><span class="title-ref"> icon to open the </span><span class="title-ref">Custom Probes</span>\` tab and create or modify a custom probe.
-> -   Click the `` `Copy Probes `` &lt;copy_probe_config><span class="title-ref"> icon to open the </span><span class="title-ref">Copy Probe</span>\` dialog, and copy the probe configurations from the currently selected object to one or more monitored objects.
+-   Click the `` `Manage Custom Probes `` &lt;pem_custom_probes><span class="title-ref"> icon to open the </span><span class="title-ref">Custom Probes</span>\` tab and create or modify a custom probe.
```
Comment (abhilasha-narendra) [inline product_docs/docs/pem/8.0/pem_online_help/04_toc_pem_features/12_pem_manage_probes/03_pem_probe_config/index.mdx:13]: Please check this on localhost if this is working as expected

File: product_docs/docs/pem/8.0/pem_online_help/04_toc_pem_features/12_pem_manage_probes/03_pem_probe_config/index.mdx [hunk 2]
```diff
@@ -10,23 +10,19 @@ A [probe](01_pem_probes/#pem_probes) is a scheduled task that returns a set of p
...
+-   Click the `` `Copy Probes `` &lt;copy_probe_config><span class="title-ref"> icon to open the </span><span class="title-ref">Copy Probe</span>\` dialog, and copy the probe configurations from the currently selected object to one or more monitored objects.
```
Comment (abhilasha-narendra) [inline product_docs/docs/pem/8.0/pem_online_help/04_toc_pem_features/12_pem_manage_probes/03_pem_probe_config/index.mdx:14]: same comment as above.

File: product_docs/docs/pem/8.0/pem_online_help/04_toc_pem_features/12_pem_manage_probes/index.mdx [hunk 3]
```diff
@@ -10,23 +10,21 @@ A [probe](03_pem_probe_config/01_pem_probes/#pem_probes) is a scheduled task tha
 
 The `Manage Probes` tab provides a set of `Quick Links` that you can use to create and manage probes:
 
-> -   Click the `` `Manage Custom Probes `` &lt;pem_custom_probes><span class="title-ref"> icon to open the </span><span class="title-ref">Custom Probes</span>\` tab and create or modify a custom probe.
-> -   Click the `` `Copy Probes `` &lt;copy_probe_config><span class="title-ref"> icon to open the </span><span class="title-ref">Copy Probe</span>\` dialog, and copy the probe configurations from the currently selected object to one or more monitored objects.
+-   Click the `` `Manage Custom Probes `` &lt;pem_custom_probes><span class="title-ref"> icon to open the </span><span class="title-ref">Custom Probes</span>\` tab and create or modify a custom probe.
```
Comment (abhilasha-narendra) [inline product_docs/docs/pem/8.0/pem_online_help/04_toc_pem_features/12_pem_manage_probes/index.mdx:13]: same comment as above

File: product_docs/docs/pem/8.0/pem_online_help/04_toc_pem_features/12_pem_manage_probes/index.mdx [hunk 4]
```diff
@@ -10,23 +10,21 @@ A [probe](03_pem_probe_config/01_pem_probes/#pem_probes) is a scheduled task tha
...
+-   Click the `` `Copy Probes `` &lt;copy_probe_config><span class="title-ref"> icon to open the </span><span class="title-ref">Copy Probe</span>\` dialog, and copy the probe configurations from the currently selected object to one or more monitored objects.
```
Comment (abhilasha-narendra) [inline product_docs/docs/pem/8.0/pem_online_help/04_toc_pem_features/12_pem_manage_probes/index.mdx:14]: same comment as above

File: product_docs/docs/pem/8.0/pem_online_help/04_toc_pem_features/12_pem_manage_probes/03_pem_probe_config/index.mdx (hunk 1 above)
Comment (nidhibhammar) [reply product_docs/docs/pem/8.0/pem_online_help/04_toc_pem_features/12_pem_manage_probes/03_pem_probe_config/index.mdx:13]: Hi Abhilasha,
I will be fixing these issues in my next PR, So please approve this.

Comment (abhilasha-narendra) [reply product_docs/docs/pem/8.0/pem_online_help/04_toc_pem_features/12_pem_manage_probes/03_pem_probe_config/index.mdx:13]: Okay

**Review by abhilasha-narendra — COMMENTED** [review]
Since Nidhi is going to fix the hyperlinks in the next check-in, we can approve and merge this one.

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**